### PR TITLE
Readme documentation completed

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone I have completed my readme.md file and also improved the indentation 
Please review this PR .

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- [Avatar](https://piston-ui.netlify.app/html-folder/avatars.html)
+- [Badge](https://piston-ui.netlify.app/html-folder/badges.html)
+- [Button](https://piston-ui.netlify.app/html-folder/buttons.html)
+- [Card](https://piston-ui.netlify.app/html-folder/cards.html)
+- [CSS Grid](https://piston-ui.netlify.app/html-folder/grids.html)
+- [Image](https://piston-ui.netlify.app/html-folder/image.html)
+- [Input](https://piston-ui.netlify.app/html-folder/inputs.html)
+- [List](https://piston-ui.netlify.app/html-folder/lists.html)
+- [Modal](https://piston-ui.netlify.app/html-folder/modal.html)
+- [Navigation]https://piston-ui.netlify.app/html-folder/nav.html)
```
Comment (nayyyhaa) [inline readme.md]: You missed one ( opening bracket here. Rest looks good. 👌

**Review by nayyyhaa — APPROVED** [review]

**Review by MayankKumar10 — APPROVED** [review]
looks good you can merge

File: readme.md (hunk 1 above)
Comment (Dev-AkshatGupta) [reply readme.md]: Thanks Neha You missed one too here :)

**Comment by Dev-AkshatGupta** [comment]
> looks good you can merge

Thank you Mayank for reviewing

**Pull request merged**